### ISSUE TITLE
Training loss suddenly increases and stays the same

### ISSUE THREAD

**Issue by gungor2** [huggingface/transformers#7524]
# ❓ Questions & Help

## Details

I am trying to develop a language model. The code is modified version of the example given [here](https://github.com/huggingface/transformers/blob/master/examples/language-modeling/run_language_modeling.py).

The problem is that at some point during the training, the training loss spikes up and stays the same. Eval loss follows this trend as well. I am attaching tensorboard plots for reference.

I am using transformers=3.2.0. I tested with transformers=3.3.0 and observed the same issue.

[TensorBoard.pdf](https://github.com/huggingface/transformers/files/5314434/TensorBoard.pdf)

**Comment by LysandreJik** [comment]
Hello, you would probably have more answers if you asked this question on the forums: https://discuss.huggingface.co

**Comment by hongjianyuan** [comment]
> # ❓ 问题与帮助
> ## 细节
> 我正在尝试开发语言模型。该代码是[此处](https://github.com/huggingface/transformers/blob/master/examples/language-modeling/run_language_modeling.py)给出的示例的修改版本。
> 
> 问题在于，在训练过程中的某个时刻，训练损失激增并保持不变。Eval loss 也遵循这一趋势。我附上张量板图以供参考。
> 
> 我正在使用变压器 = 3.2.0。我使用transformers=3.3.0 进行了测试并观察到了同样的问题。
> 
> [TensorBoard.pdf](https://github.com/huggingface/transformers/files/5314434/TensorBoard.pdf)

I also encountered this problem, is it solved?

**Comment by KatarinaYuan** [comment]
Hi, I encountered similar issue when pretraining BERT. Have you solved this problem? Could you please share some insights?
